kyototycoon/tests/test_kyototycoon.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.kyototycoon import KyotoTycoonCheck
+
+from .common import (
+    URL
+)
+
+
+@pytest.mark.integration
+def test_check(aggregator, kyototycoon):
+    """
+    Testing Kyototycoon check.
+    """
+    kt = KyotoTycoonCheck('kyototycoon', {}, {})
+
+    TAGS = ['optional:tag1']
+
+    instance = {
+        'report_url': '{0}/rpc/report'.format(URL),
+        'tags': TAGS
+    }
+
+    # run the check twice so we can get some rate metrics
+    for i in range(2):
+        kt.check(instance)
```

Comment:
No need to run that twice since, as you said below, the aggregator doesn't compute rates.